From 93de63cac7192cde1fcfa880c22614039d04f220 Mon Sep 17 00:00:00 2001
From: Adeel Arshad <adeel_arshad@mentor.com>
Date: Tue, 18 Oct 2016 18:52:00 +0500
Subject: [PATCH] remove remote agent dependency on UI components

CXLRemoteDebuggingServer is dependent on the FrameworkComponents,
and FrameworkComponents also includes the UI components that are
not required for remote agent so remove the UI components from
FrameworkComponents for the time being. Proper fix would be divide
the FrameworkComponents into two subset, one will include the
non UI components and second will include the UI components.

Signed-off-by: Adeel Arshad <adeel_arshad@mentor.com>
---
 CodeXL/SConstruct | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CodeXL/SConstruct b/CodeXL/SConstruct
index e742ad3..d469d98 100755
--- a/CodeXL/SConstruct
+++ b/CodeXL/SConstruct
@@ -185,7 +185,7 @@ FrameworkComponents += Application_Obj
 # This depends upon everything built to date
 Application_Obj = SConscript('App/SConscript', variant_dir=obj_variant_dir+'/AMDCodeXL', duplicate=0)
 CXL_env.Depends(Application_Obj, FrameworkComponents)
-FrameworkComponents += Application_Obj
+#FrameworkComponents += Application_Obj
 
 SharedProf_Obj = SConscript('Components/AMDTSharedProfiling/SConscript', variant_dir=obj_variant_dir+'/AMDTSharedProfiling', duplicate=0)
 CXL_env.Depends(SharedProf_Obj, FrameworkComponents)
-- 
1.9.1

